setup.py

```diff
@@ -350,73 +419,54 @@ def run(self):
         self._compile_webhost(self.webhost_dir)
 
 
-with open("README.md") as readme:
-    long_description = readme.read()
+class Clean(distutils.cmd.Command):
+    description = 'Clean up build generated files'
+    user_options = []
+
+    def __init__(self, dist: Distribution):
+        super().__init__(dist)
+        self.dir_list_to_delete = [
+            "build"
+        ]
+
+    def initialize_options(self) -> None:
+        pass
+
+    def finalize_options(self) -> None:
+        pass
+
+    def run(self) -> None:
+        for dir_to_delete in self.dir_list_to_delete:
+            dir_delete = pathlib.Path(dir_to_delete)
+            if dir_delete.exists():
+                dir_util.remove_tree(str(dir_delete))
+
+
+COMMAND_CLASS = {
+    'develop': Development,
+    'build': BuildProtos,
+    'webhost': Webhost,
+    'extension': Extension,
+    'clean': Clean
```

Comment:
New clean to remove `build`. Can be extended in the future to make it remove other stuff.